girder_tulane/web_client/package.json

```diff
@@ -0,0 +1,154 @@
+{
+    "name": "@girder/girder_tulane",
+    "version": "0.1.0",
```

Comment:
If we are using scm, this should just be 0.0.0

Reply:
See 612d1471779c9b491daa4bae24593f63ea1b840b